src/button/index.ts

```diff
@@ -0,0 +1,3 @@
+export { Button, useButton } from "reakit/Button";
```

Comment:
`AriaButton` uses @react-aria/button right? why we are exporting reakit's button in here?

Reply:
I exported it thinking that we are building a separate library.
Instead of using "reakit/Button" when using our components to build our UI.
I thought we export all the reakit components and use only from this repo, this way user need to only install our library.
Just a reexport and importing it again.

Reply:
Okay got it! fair enough.